components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -28,7 +28,10 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
         canViewSystemErrors: PropTypes.bool.isRequired,
         totalUsers: PropTypes.number,
         dismissedExpiringLicense: PropTypes.bool,
+        dismissedNumberOfActiveUsersWarnMetricStatus: PropTypes.bool,
         siteURL: PropTypes.string.isRequired,
+        dismissWarnMetricStatus: PropTypes.bool,
```

Comment:
I think this property is not used or passed. Probably something part of the development process.

Reply:
yes, it is a leftover, will clean up.